oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -325,6 +339,10 @@ public void execute(String... args) throws Exception {
             } else {
                 System.err.println("unknown revisions command: " + subCmd);
             }
+            if (exitWhenDone) {
```

Comment:
this change is causing the build to fail locally because it exits the running JDK instance.
cc @Joscorbe @daniancu